doc/source/whatsnew/v1.3.3.rst

```diff
@@ -17,7 +17,7 @@ Fixed regressions
 - Fixed regression in :class:`DataFrame` constructor failing to broadcast for defined :class:`Index` and len one list of :class:`Timestamp` (:issue:`42810`)
 - Performance regression in :meth:`core.window.ewm.ExponentialMovingWindow.mean` (:issue:`42333`)
 - Fixed regression in :meth:`.GroupBy.agg` incorrectly raising in some cases (:issue:`42390`)
--
+- Fixed regression in :meth:`core.dtypes.cast.convert_dtypes` incorrectly converts byte strings to strings (:issue:`43183`)
```

Comment:
this is an internal function, instead you want to use `DataFrame.convert_dtypes` which is the user facicng one.

Reply:
Done